src/scripts/main.js

```diff
@@ -1 +1,91 @@
 'use strict';
+
+const success = 'rgba(10, 189, 0, 0.3)';
+const error = 'rgba(253, 0, 0, 0.3)';
+
+const promise1 = new Promise((resolve, reject) => {
```

Comment:
```suggestion
const firstPromise = new Promise((resolve, reject) => {
```
не використовуй цифри в іменах змінних